x-pack/plugins/monitoring/public/directives/alerts/index.js

```diff
@@ -54,17 +54,17 @@ const alertRowFactory = (scope, kbnUrl) => {
 
     return (
       <KuiTableRow>
-        <KuiTableRowCell>
+        <KuiTableRowCell tabIndex="0">
```

Comment:
Does each cell really need to be tabbable? Is this just to make it visible to a screen-reader?

Reply:
I'd imagine so, otherwise the screen reader can't read any of the content in non tabbable cells

Reply:
I think I've found this to be true myself. I was hoping there'd be another way :D